README.md

```diff
@@ -10,6 +10,16 @@ npm install @inrupt/solid-client solid-auth-client
 
 See https://inrupt.github.io/solid-client-js/docs/
 
+# Related libraries
+
+## Authentication
+
+To send authenticated requests to your Pod and retrieve private data, you can use [solid-client-authn-browser](https://github.com/inrupt/solid-client-authn).
+
+## Vocabularies and interoperability
+
+To reuse well-known vocabularies, and make your app produce/consume data in a more interoperable way, we provide vocabularies available as constants that you just have to import: [solid-common-vocab-rdf](https://github.com/inrupt/solid-common-vocab-rdf).
```

Comment:
Ditto - although the package name here does not seem to match the repository?

```suggestion
To reuse well-known vocabularies, and make your app produce/consume data in a more interoperable way, we provide vocabularies available as constants that you just have to import: [solid-common-vocab-rdf](https://www.npmjs.com/package/@inrupt/solid-common-vocab).
```

Reply:
In this case, there are multiple NPM packages available, so I think linking to those in the `https://github.com/inrupt/solid-common-vocab-rdf` README, with a description of their content, might be helpful and avoid duplicating the info there, or just linking to multiple packages without a description.

Reply:
Ah, and they're all generated from the same repo? Shouldn't it have its own website then?

But yeah, fine with leaving it pointing to GitHub then :)

Reply:
Yeah we had a tutorial for using these vocabylary artifacts, eventually I agree it should have its own website :)